docs/_includes/include-lines-tags.adoc

```diff
@@ -88,9 +89,37 @@ snippet b
 
 Notice that none of the lines with the tag directives are displayed.
 
-NOTE: for XML files you can use the `+<!-- tag::name[] -->+` and `+<!-- end::name[] -->+` delimiters
+==== By expressions using tags
 
-Alternately, you can select content by line number.
+The previous section showed how to select tagged regions explictly, but you can also use expressions to select groups of tags:
+
+[horizontal]
+`*`::
+  All tagged regions
+
+`**`::
+ Non-tagged regions
+
+`!`::
+ Exclude
+
+The inclusive operations (explicit names, `\*` and `**`) are applied first, then the exclusive operations (`!`), so it doesn't matter which order they appear in.
+
+Some examples:
+
+`tags=a;!b` selects regions tagged a but excludes any _nested_ regions tagged b.
```

Comment:
Perhaps use "foo" and "bar" here instead of "a" and "b" so it's easier to see when you are referring to a tag name.